module-2/backend/week_three.md

```diff
@@ -10,14 +10,37 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+`rails new <project_name> -T -d postgresql --skip-spring --skip-turbolinks`
+
 2. What do Models generally inherit from in rails?
+>ApplicationRecord
+
 3. What do Controllers generally inherit from in a rails project?
-4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
-5. What rake task is useful when looking at routes, and what information does it give you?
-6. What is an example of a route helper? When would you use them?
-7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+>ApplicationController
+
+4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality? `resources :horses, only: [:show]`
+
+5. What rake task is useful when looking at routes, and what information does it give you? `rake routes`
+
+6. What is an example of a route helper? When would you use them? `company_job_path` I would it when I want to redirect or link to somewhere else. ie `<%= link_to "Edit", edit_company_job_path(job.company, job) %>`
+
+7. What's the difference between what `_url` and `_path` return when combined with a routes prefix? 
+>Don't they return the same thing?
+
 8. What are strong params and why are the necessary?
+> Thdey are needed to limit what the client is allowed to access when CRUDing.
+
 9. What role does `form_for` play in helping us create our forms?
+> It allows us to have basic forms to fill in when we want to access some of the CRUD functionality in our app.  
+
 10. How does `form_for` know where to submit the user's input?
+>Because RAILS is magic. Honestly I think it's because when the request comes in, it is paired with the appropriate action.
```

Comment:
We provide it with instance variables that interpret to a path. In our routes, we have told our route where to `POST` to and we create a route that mimics that with instance variables in the view